internal/dag/dag.go

```diff
@@ -636,6 +640,41 @@ type SecureVirtualHost struct {
 	// AuthorizationServerWithRequestBody specifies configuration
 	// for buffering request data sent to AuthorizationServer
 	AuthorizationServerWithRequestBody *AuthorizationServerBufferSettings
+
+	// JWTProviders specify how to verify JWTs.
+	JWTProviders []JWTProvider
+}
+
+type JWTProvider struct {
+	Name       string
+	Issuer     string
+	Audiences  []string
+	RemoteJWKS RemoteJWKS
+}
+
+type RemoteJWKS struct {
+	HTTPURI       HTTPURI
```

Comment:
can do the same inlining as with the API here, the nesting is not useful for us

Reply:
👍🏽